src/Payum/Offline/Action/FillOrderDetailsAction.php

```diff
@@ -16,6 +16,10 @@ class FillOrderDetailsAction implements ActionInterface
     public function execute($request)
     {
         RequestNotSupportedException::assertSupports($this, $request);
+        
+        if($request->getToken()->getDetails()) {
+            return;
+        }
```

Comment:
I think that is not case, cause payment is not completed until you receive money do basically paid is false until then